libs/net/models/Settings/ReportSectionSettingsModel.cs

```diff
@@ -51,6 +53,7 @@ public ReportSectionSettingsModel(JsonDocument settings, JsonSerializerOptions o
         this.HideEmpty = settings.GetElementValue("hideEmpty", false, options)!;
         this.GroupBy = settings.GetElementValue("groupBy", "", options)!;
         this.SortBy = settings.GetElementValue("sortBy", "", options)!;
+        this.ShowContent = settings.GetElementValue("showContent", false, options);
```

Comment:
What is this option for?

Reply:
Same as first comment above.

Reply:
I don't understand what `ShowContent` is compared to `ShowFullStory`?